var/spack/repos/builtin/packages/py-distance/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyDistance(PythonPackage):
+    """This package provides helpers for computing similarities
+    between arbitrary sequences. Included metrics are
+    Levenshtein, Hamming, Jaccard, and Sorensen distance, plus
+    some bonuses."""
+
+    homepage = "https://github.com/doukremt/distance"
+    pypi     = "Distance/Distance-0.1.3.tar.gz"
+
+    version('0.1.3', sha256='60807584f5b6003f5c521aa73f39f51f631de3be5cccc5a1d67166fcbf0d4551')
```

Comment:
You'll want to add a build dep on setuptools. We use pip now to build packages, and pip silently replaces distutils with setuptools when it builds a package.